.circleci/config.yml

```diff
@@ -2,10 +2,10 @@
 #  Each post contains a full image manifest, including iOS runtimes, devices, CocoaPods version, etc.
 #  NOTE: When updaing Xcode check the manifest for compatible chruby versions.
 anchors:
-  - &android-env     "cimg/android:2023.08.1-node"
-  - &latest-xcode    "14.3.1"
-  - &latest-ios      "16.4"
-  - &min-ios         "15.5"
+  - &android-env     "cimg/android:2023.09.1-node"
+  - &latest-xcode    "15.0.0"
+  - &latest-ios      "17.0"
+  - &min-ios         "16.4"
```

Comment:
Should the min ios go up now or in 12.0 ?

Reply:
The min will go up in 12.  The issue is that the images only contain have 2 iOS versions.   

Before:
- xcode 14 default for iOS 15.5 and 16.4 testing
- xcode 15 additional build for iOS 17 testing

After:
- xcode 15 default for iOS 16.4 and 17.0 testing
- xcode 14 additional build for 15.5 testing  <- we can drop this without code changes when MSDK 12 goes out.

Reply:
👍